app/controllers/follow_requests_controller.rb

```diff
@@ -23,10 +23,11 @@ def edit
   # POST /follow_requests or /follow_requests.json
   def create
     @follow_request = FollowRequest.new(follow_request_params)
+    @follow_request.sender = current_user
```

Comment:
You could leverage an association here to combine lines

```suggestion
    @follow_request = current_user.follow_requests.new(follow_request_params)
```

Reply:
That makes sense.  Thanks.